packages/dtslint/src/index.ts

```diff
@@ -21,6 +21,7 @@ async function main(): Promise<void> {
   let dirPath = process.cwd();
   let onlyTestTsNext = false;
   let expectOnly = false;
+  let npmNamingOnly = false;
```

Comment:
Ah, I think I should have described this better. This check should still be non-optional and not have a flag; it's just that it should be enabled via the overrides passed to the ESLint class, rather than enabled via a preset. No need to make a flag or anything.